src/utils/logToFile.ts

```diff
@@ -10,8 +10,4 @@ electronLog.transports.file.maxSize = 2 * 1024 * 1024;
 // one file in stand of main.log & renderer.log
 // electronLog.transports.file.fileName = 'logs.log';
 
-if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') {
```

Comment:
Logs are not visible in the production environment too, Either set NODE_ENV or we should remove this block